src/common.h

```diff
@@ -47,6 +47,9 @@
 
 #define ARRAY_SIZE(x) (sizeof(x) / sizeof(x[0]))
 
+#define PROF_STRINGIFY_(n) #n
```

Comment:
I don't understand what this is doing.

Reply:
https://gcc.gnu.org/onlinedocs/cpp/Stringizing.html